include/seqan3/alignment/pairwise/alignment_configurator.hpp

```diff
@@ -168,48 +132,49 @@ struct alignment_configurator
 private:
 
     /*!\brief Transformation trait that chooses the correct matrix policy.
-     * \tparam config_type The configuration for which to select the correct matrix policy.
-     * \tparam score_t The value type used for the score matrix.
-     * \tparam trace_t The value type used for the trace matrix.
+     * \tparam traits_t The alignment configuration traits type.
      */
-    template <typename config_type, typename score_t, typename trace_t>
+    template <typename traits_t>
     struct select_matrix_policy
     {
     private:
+        //!\brief Indicates whether only the coordinate is required to compute the alignment.
```

Comment:
```suggestion
        //!\brief Indicates whether only the end coordinate is required to compute the alignment.
```

Reply:
well, actually both. To compute the front coordinate we actually don't need a traceback but it would suffice to compute the reverse alignment starting from the found end coordinate. This could also be done in a banded fashion and thus can be faster. So the sentence implies that only the coordinates are tracked and not the entire trace matrix.

Reply:
Okay then I suggest
```suggestion
        //!\brief Indicates whether only the coordinates are required to compute the alignment.
```